collect_app/src/main/res/layout/quit_form_dialog_layout.xml

```diff
@@ -0,0 +1,75 @@
+<?xml version="1.0" encoding="utf-8"?>
+<androidx.constraintlayout.widget.ConstraintLayout xmlns:android="http://schemas.android.com/apk/res/android"
+    xmlns:app="http://schemas.android.com/apk/res-auto"
+    android:layout_width="match_parent"
+    android:layout_height="wrap_content"
+    android:padding="@dimen/margin_large">
+
+    <com.google.android.material.textview.MaterialTextView
+        android:id="@+id/save_explanation"
+        android:layout_width="0dp"
+        android:layout_height="wrap_content"
+        android:text="@string/save_explanation"
+        android:textAppearance="?textAppearanceBodyMedium"
+        app:layout_constraintEnd_toEndOf="parent"
+        app:layout_constraintStart_toStartOf="parent"
+        app:layout_constraintTop_toTopOf="parent" />
+
+    <com.google.android.material.button.MaterialButton
+        android:id="@+id/discard_changes"
+        style="@style/Widget.Material3.Button.OutlinedButton.Icon"
+        android:layout_width="0dp"
+        android:layout_height="match_parent"
+        android:layout_marginTop="@dimen/margin_extra_large"
+        android:text="@string/discard_changes"
+        app:icon="@drawable/ic_delete"
+        app:iconGravity="textStart"
+        app:iconSize="18dp"
+        app:layout_constraintEnd_toEndOf="parent"
+        app:layout_constraintStart_toStartOf="parent"
+        app:layout_constraintTop_toBottomOf="@id/save_explanation" />
+
+    <com.google.android.material.button.MaterialButton
+        android:id="@+id/keep_editing_outlined"
+        style="@style/Widget.Material3.Button.OutlinedButton.Icon"
+        android:layout_width="0dp"
+        android:layout_height="match_parent"
+        android:layout_marginTop="@dimen/margin_small"
+        android:text="@string/keep_editing"
+        app:icon="@drawable/ic_edit"
+        app:iconGravity="textStart"
+        app:iconSize="18dp"
+        app:layout_constraintEnd_toEndOf="parent"
+        app:layout_constraintStart_toStartOf="parent"
+        app:layout_constraintTop_toBottomOf="@id/discard_changes" />
+
+    <com.google.android.material.button.MaterialButton
```

Comment:
I was surprised I'd never run into this before, but it's currently not possible to change the style of a view programmatically (or after it's inflated more accurately). This means having duplicated views for the "Keep editing" button.